islandora.module

```diff
@@ -611,6 +612,10 @@ function islandora_permission() {
       'title' => t('Revert datastream history'),
       'description' => t('Revert to a previous version of a datastream.'),
     ),
+    ISLANDORA_ACCESS_INACTIVE_AND_DELETED_OBJECTS => array(
```

Comment:
If this permission is not held, but `ISLANDORA_MANAGE_PROPERTIES` _is_, it would be possible for users to change the state away from `Active` and immediately lose access to the object. These kinds of behaviour we generally try to avoid (like in XACML, we ensure the current user is present in the `POLICY` on form submission), or make very explicit with confirmation forms and the like.